.github/workflows/ci.yaml

```diff
@@ -212,7 +219,9 @@ jobs:
       - name: Build the source and binary distribution
         run: python -m build
 
+        # It is sufficient to upload only once
       - name: Upload distribution files
+        if: ${{ matrix.os == 'ubuntu-20.04' }}
```

Comment:
This is not strictly true, we should also test that the built package is installable (we do it in the `test-installation` job. Not sure if we really need to do this using QEMU and docker though, for archs directly supported by GitHub it might be much faster to just do it installing in a venv.